scripts/src/release/releaser.py

```diff
@@ -0,0 +1,160 @@
+"""
+Used within github actions, specifically as part of the charts auto release process defined in
+.github/workflow/release.yml. Makes all of the changes required to both the chart and development repos.
+
+
+parameters :
+  --version : the Version of the the release being created.
+  --pr_dir : the directory containing the PR contents. Used to get the release-info.jso file.
+  --development_dir : the directory containing the latest version of the development repository.
+  --charts_dir : the directory containing the latest version of the charts repository.
+
+Performs these action.
+- Gets a list of updates to perform from the pr_dir releases/release_info.json file. These updates are then made
+to the charts and development repositories.
+- Adds the cron job to .github/worklfows/schedule.yml and changes the verifier image used in .github/worklfows/schedule.yml
+  to latest, as required. The charts repo is updated from development repo which necessitates these update.
+- Create a PR against the charts repo containing the workflow updates. This requires manual merge.
+- Directly commits to the development main branch any new charts added to the charts repo since the last update.
+
+
+"""
+import yaml
+import os
+import argparse
+import sys
+from release import release_info
+
+sys.path.append('../')
+from github import gitutils
+
+SCHEDULE_YAML_FILE=".github/workflows/schedule.yml"
+BUILD_YAML_FILE=".github/workflows/build.yml"
+
+SCHEDULE_INSERT = [
+    '  # Daily trigger to check updates',
+    '  schedule:',
+    '    - cron: "0 0 * * *"'
+]
+
+def update_workflow():
+
+    lines=[]
+    with open(SCHEDULE_YAML_FILE,'r') as schedule_file:
+
+        lines = schedule_file.readlines()
+
+        for line in lines:
+            if line.strip() == "on:":
+                insert_location = lines.index(line)+1
+                if SCHEDULE_INSERT[0] not in lines[insert_location].rstrip():
+                    print("[INFO] add cron job to schedule.yaml")
+                    lines.insert(insert_location,f"{SCHEDULE_INSERT[0]}\n")
+                    lines.insert(insert_location+1,f"{SCHEDULE_INSERT[1]}\n")
+                    lines.insert(insert_location+2,f"{SCHEDULE_INSERT[2]}\n")
```

Comment:
I looked at jinja and it appeared to be for html and not yml and looked like it would be hard for me to figure it out so I went for cheap and cheerful. My plan was after I wrote cheap and cheerful I would  go back and look at jinja but then I noticed the code was failing badly in other places and got consumed in fixing that and not looking back a jinja. 
Up to you on this, can go back and look at jinja if you prefer.

Reply:
leave like this if it works, I can change this piece once it is merged

Reply:
Sounds like a plan.